File: extract/py/testdata/greeter/greeter.py
"""Greeter greets by name."""


class Greeter:
    """Greeter greets by name."""

    def __init__(self, name: str) -> None:
        self.name = name

    def greet(self) -> str:
        """greet returns this Greeter's greeting."""
        return "hello, " + self.name

File: extract/ts/testdata/greeter/greeter.ts
// Greeter greets by name.
export class Greeter {
  name: string;

  constructor(name: string) {
    this.name = name;
  }

  // greet returns this Greeter's greeting.
  greet(): string {
    return "hello, " + this.name;
  }
}
